src/Components/MinimalCtaBanner.tsx

```diff
@@ -1,55 +1,88 @@
 import React from "react"
 import styled from "styled-components"
 import { unica } from "../Assets/Fonts"
+import SlideTransition from "./Animation/SlideTransition"
 import Icon from "./Icon"
 
 export interface MinimalCtaBannerProps extends React.Props<HTMLDivElement> {
   backgroundColor?: string
   copy?: string
+  height?: string
   href?: string
   position: "top" | "bottom"
   textColor?: string
+  show?: boolean
 }
 
-const MinimalCtaBanner: React.SFC<MinimalCtaBannerProps> = props => (
-  <BannerContainer
-    position={props.position}
-    backgroundColor={props.backgroundColor}
-  >
-    <Banner href={props.href} textColor={props.textColor}>
-      <p>{props.copy}</p>
-      <IconContainer>
-        <Icon name="chevron-right" color={props.textColor} fontSize="12px" />
-      </IconContainer>
-    </Banner>
-  </BannerContainer>
-)
+export interface State {
+  dismissed: boolean
+}
+
+export class MinimalCtaBanner extends React.Component<
+  MinimalCtaBannerProps,
+  State
+> {
+  state = {
+    dismissed: false,
+  }
+
+  dismissCta = () => {
+    this.setState({ dismissed: true })
+  }
+
+  render() {
+    const ctaBanner = (
+      <SlideTransition
+        in={this.props.show}
+        timeout={{ enter: 10, exit: 250 }}
+        height={this.props.height}
```

Comment:
I wonder if there is a way to default the `SlideTransition`'s height to `min-content` or `inherit` and avoid this prop altogether

Reply:
I chatted about this IRL with @l2succes and @yuki24. They thought it might be fine to have a specified height for the time being. Let me know if you have strong opinions about this because I would totally be open to more dialogue.

Reply:
Ok I think worth revisiting when the artist CTA is implemented.